schema_metadata_templates/shared/assay_STARRSeq_metadata_template.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.metadataTemplates-assay.STARRSeq",
+    "description": "SysBio STARRSeq metadata template schema",
+    "properties": {
+        "specimenID": {
+            "$ref": "sage.annotations-experimentalData.specimenID"
+        },
+        "libraryID": {
+            "$ref": "sage.annotations-ngs.libraryID"
+        },
+        "assay": {
+          "$ref": "sage.annotations-experimentalData.assay"
+        },
+        "platform": {
+            "$ref": "sage.annotations-experimentalData.platform"
+        },
+        "RIN": {
+            "$ref": "sage.annotations-qc.RIN"
+        },
+        "dnaBatch": {
+            "$ref": "sage.annotations-batch.dnaBatch"
+        },
+        "referenceSet": {
+            "$ref": "sage.annotations-curatedData.referenceSet"
+        },
+        "rnaBatch": {
+            "$ref": "sage.annotations-batch.rnaBatch"
+        },
+        "libraryBatch": {
+            "$ref": "sage.annotations-batch.libraryBatch"
+        },
+        "sequencingBatch": {
+            "$ref": "sage.annotations-batch.sequencingBatch"
+        },
+        "libraryPrep": {
+            "$ref": "sage.annotations-ngs.libraryPrep"
+        },
+        "libraryPreparationMethod": {
+            "$ref": "sage.annotations-ngs.libraryPreparationMethod"
+        },
+        "libraryVersion": {
+            "$ref": "sage.annotations-ngs.libraryVersion"
+        },
+        "isStranded": {
+            "$ref": "sage.annotations-ngs.isStranded"
+        },
+        "readStrandOrigin": {
+            "$ref": "sage.annotations-ngs.readStrandOrigin"
+        },
+        "readLength": {
+            "$ref": "sage.annotations-ngs.readLength"
+        },
+        "runType": {
+            "$ref": "sage.annotations-ngs.runType"
+        },
+        "totalReads": {
+            "$ref": "sage.annotations-experimentalData.totalReads"
+        },
+        "validBarcodeReads": {
+            "$ref": "sage.annotations-qc.validBarcodeReads"
+        },
+        "numberCells": {
+            "$ref": "sage.annotations-experimentalData.numberCells"
+        },
+        "medianGenes": {
+            "$ref": "sage.annotations-qc.medianGenes"
+        },
+        "medianUMIs": {
+            "$ref": "sage.annotations-qc.medianUMIs"
+        },
+        "ratio260over280": {
+            "$ref": "sage.annotations-qc.ratio260over280"
+        },
+        "ratio260over230": {
+            "$ref": "sage.annotations-qc.ratio260over230"
+        },
+        "GQN": {
+            "$ref": "sage.annotations-qc.GQN"
+        }
+    }
+}
```

Comment:
Does it need an empty line after the last bracket? I think that's the case for the mini schemas in synapseAnnotations but I can't remember if that applies here.

Reply:
I don't see an empty line in the other templates!